pages/Getting Started/Master-Tutorial.md

```diff
@@ -47,7 +47,8 @@ Please bear in mind 3D acceleration in VMs may be pretty slow.
 
 Hyprland can be executed by typing `Hyprland` in your tty.
 
-Systemd users can also start Hyprland, by typing `systemctl --user start hyprland-session.service` in tty. This method provides additional features, such as [xdg-autostart](https://www.freedesktop.org/software/systemd/man/latest/systemd-xdg-autostart-generator.html) support and ability to enable services for programs that rely on a graphical session and provide such services (e.g waybar).
+Systemd users can also start Hyprland, using [uwsm](https://github.com/Vladimir-csp/uwsm). This is the recommended method of launching Hyprland on systemd-based distros, as it provides additional features, such as [xdg-autostart](https://www.freedesktop.org/software/systemd/man/latest/systemd-xdg-autostart-generator.html) support and the ability to enable services for programs that rely on a graphical session and provide such services (e.g waybar). See [uwsm](../../Systemd-start) page for further instructions.
```

Comment:
...plus manage session all-apps-as-units way.

Reply:
Regarding that, I see a lot of `\x2d` escapes in the generated units. Is this the fault of NixOS in some way?
![image](https://github.com/user-attachments/assets/fb70a850-ac73-419e-8e0d-92a45eea3cb5)

Reply:
Believe it or not, this is how it should be. `-` is a special symbol in unit IDs, it should be escaped when not intended for globs. See `man systemd-escape`

Reply:
Do programs have to specify `WantedBy = wayland-session@hyprland\x2duwsm.desktop.target` then? Or does the unescaped name also work?

Although the xdg-autostart route may be the idiomatic way.

Reply:
`find /run/user/${UID}/systemd/ -name '*\x*'`. Yes, this is canonical representation, also in filenames.